concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -486,6 +635,13 @@ local ImgGroup(name, images) = {
           for env in envs
           for image in rhel_images
         ] +
+        [
+          // SAP related test jobs
+          saptestjob {
+            image: image
+          }
+          for image in std.filter(function(x) std.endsWith(x, '-sap'), rhel_images)
```

Comment:
just list them out rather than this kind of thing, i can't tell what it will be by reading. if it's long, create a list with `local sap_images = [image,image];`